### DISCUSSION TITLE
Wasabi as storage doesn't seem to work

### DISCUSSION THREAD

**Discussion by kassuro** [appwrite/appwrite#4278] [Q&A]
Hello everyone,

today I tried to connect appwrite to a wasabi bucket as storage, but what ever I do, it seems nothing is stored in the actual bucket but instead on local storage?

I even tried it with a complete blank installation of v1. Changed the `.env` file like this:
```
_APP_STORAGE_DEVICE=Wasabi
_APP_STORAGE_WASABI_ACCESS_KEY=MyAccessKey
_APP_STORAGE_WASABI_SECRET=MySecret
_APP_STORAGE_WASABI_REGION=eu-central-1
_APP_STORAGE_WASABI_BUCKET=nwstest2
```

Then killed the whole installation with `docker compose down -v` and recreated everything with `docker compose up -d`. Created a new project, new bucket, uploaded a file. Was uploaded successful but no file in the bucket.

Do I miss something? Where could I find information where it goes wrong? `docker logs appwrite` doesn't provide any errors for the bucket or wasabi connection.

Edit: made a copy paste error with the env variable names. I used the correct ones in the `.env` file

**Comment by stnguyen90** [comment]
`_APP_STORAGE_DEVICE` looks fine, but for the others, can you try:

```
_APP_STORAGE_WASABI_BUCKET=bucket name
_APP_STORAGE_WASABI_REGION=region
_APP_STORAGE_WASABI_SECRET=my secret
_APP_STORAGE_WASABI_ACCESS_KEY=my access key
```

**Reply by kassuro** [reply]
> `_APP_STORAGE_DEVICE` looks fine, but for the others, can you try:
> 
> ```
> _APP_STORAGE_WASABI_BUCKET=bucket name
> _APP_STORAGE_WASABI_REGION=region
> _APP_STORAGE_WASABI_SECRET=my secret
> _APP_STORAGE_WASABI_ACCESS_KEY=my access key
> ```

Oh my bad, that's a copy/paste error in my question. I have the correct ones in the `.env` file. I updated my question with the correct variable names

**Comment by TenUchikawa** [comment]
I'm having same issue
